pdf2remarkable.sh

```diff
@@ -70,28 +70,74 @@ REMARKABLE_HOST=${REMARKABLE_HOST:-remarkable}
 REMARKABLE_XOCHITL_DIR=${REMARKABLE_XOCHITL_DIR:-.local/share/remarkable/xochitl/}
 TARGET_DIR="${REMARKABLE_HOST}:${REMARKABLE_XOCHITL_DIR}"
 
+# Function to show help
+show_help ()
+{
+    echo "Transfer PDF or EPUB document(s) to a reMarkable tablet."
+    echo "usage: $(basename $0) [options] path-to-file [path-to-file]..."
+    echo "  -h  --help             print this usage information and exit"
+    echo "  -q  --quiet            don't print progress information"
+    echo "  -r  --toggle-restart   toggle whether to restart the tablet (default is given by RESTART_XOCHITL_DEFAULT)"
```

Comment:
"toggle-restart" and "restart the tablet" could be confusing/misleading, since `systemctl restart xochitl` is just restarting the UI service and not rebooting the tablet.  for many people "restarting" a device is a reboot or power cycle.

Reply:
Fair point. What do you suggest instead? Saying "restart the tablet's UI" would be accurate but unenlightening, as it does not explain the purpose behind the restart. One could say "refresh the document database on the tablet" but does not explain that the whole UI will refresh.

Reply:
i'm just doing a drive-by comment since it's really up to @adaerr -- but FWIW i'd say that rather be technically-correct (but possibly confusing), be user-focused and use terminology that explains the effects visible to the user rather than how it's accomplished.  so i suggest something like `--make-visible` or `--show-docs-now` or something along those lines, and the extended help comment would say "make the transferred document visible in the UI now (rather than after next reboot)" or something along those lines.

Reply:
Thanks for the suggestions. I feel that it would still be quite weird to have `-r` short options to expand to `--toggle-make-visible`, and the latter is quite a mouthful. I leave to @adaerr to make the final call.